reference/request-bodies/payment_service_update_request.yaml

```
@@ -169,3 +169,19 @@ properties:
     description: |-
       Defines if tokenization is enabled for the service (can only be
       enabled if the payment service definition supports it).
+
+  network_tokens_enabled:
+    type: boolean
+    example: true
+    nullable: true
+    description: |-
+      Defines if network tokens are enabled for the service (can only be
+      enabled if the payment service definition supports the
+      `network_tokens` feature).
+
+      If this value is not provided, it will be set to the value of
```

```suggestion
      If this value is not provided or is `null`, it will be set to the value of
```

(please check linting, it may require to wrap the line 🙏 )

---

Done: https://github.com/gr4vy/gr4vy-openapi/pull/238/commits/a04641804aa79578bbf50fd5d550cf5ca49f05cf